app/controllers/games_controller.rb

```diff
@@ -61,4 +65,8 @@ def set_player
       redirect_to new_game_player_path(game_slug: slug_param) if @player.blank?
     end
   end
+
+  def set_game
+    @game = Game.find_by(slug: slug_param) or return redirect_to games_not_found_path(slug: slug_param)
```

Comment:
I *think* we can drop the `return` here now, because we are on the last line of a method, so it will be returned anyway.